.github/workflows/ui.yml

```diff
@@ -0,0 +1,48 @@
+name: UI Tests
+
+on: [push]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup conda
```

Comment:
Why do we need conda? Does this not run in just python? I would like this file to be similar to `test.yml`